### ISSUE TITLE
W036 false positive: a FOR /F command whose text contains "file" (e.g. `powershell -NoProfile`) is treated as a data file

### ISSUE THREAD

**Issue by fametrano** [tboy1337/Blinter#37]
## What happens

W036 ("FOR /F missing skip option for headers") fires on a `FOR /F` loop that runs a command, not a file, whenever the command text contains the substring `file`. `-NoProfile` contains it, so every loop that captures PowerShell output with `-NoProfile` gets the warning.

## Minimal sample

Only the relevant lines of Blinter's output are shown below (S007 and similar are left out).

`test.bat`:

```bat
@echo off
for /f "usebackq delims=" %%V in (`powershell -NoProfile -Command "1+1"`) do set "D=%%V"
exit /b 0
```

```text
> blinter test.bat --no-config --summary
Line 2: FOR /F missing skip option for headers (W036)
```

Controls (each is the same file with one change; none reports W036):

- `-NoProfile` shortened to `-NoPro` (PowerShell accepts it): no W036
- `-NoProfile` removed: no W036
- `` `ver` `` as the command: no W036

And the reverse: `` `myprofiler.exe --list` `` as the command reports W036, because the program name contains `file`.

## Expected

No W036. The operand is a backquoted command under `usebackq`, so there is no file and no header line. Following the recommendation (`skip=1`) would throw away the first line of the command's output, which in this sample is the whole output.

## Where it comes from

`blinter/checkers/advanced/vars_syntax.py`, W036 block: the test is `"file" in file_operand.lower()`, a substring match over the whole `in (...)` text. `_for_f_file_set_operand` only excludes single-quoted operands, so a backquoted command goes through.

A possible direction: under `usebackq`, skip the rule for a backquoted operand (it is a command); without `usebackq`, skip it for a single-quoted one.

## Environment

Blinter v1.1.23 (also v1.1.21), Python 3.14.6, macOS 26.6.2, run via `uvx blinter`.

**Comment by tboy1337** [comment]
Fixed in v1.1.24 (https://github.com/tboy1337/Blinter/releases/tag/v1.1.24, commit f4d42ab).

W036 now classifies FOR /F IN (...) operands using cmd.exe rules: usebackq backticks are commands, not data files. powershell -NoProfile no longer triggers a missing skip= warning.

CI for this release is green: https://github.com/tboy1337/Blinter/actions/runs/34097885302
